src/app/features/courses/courses-list/courses-list.component.html

```diff
@@ -0,0 +1,9 @@
+<main class="course-list">
+  <app-course-card
+    *ngFor="let course of courses"
+    [course]="course"
+    (showCourse)="onShowCourse($event)">
+    <app-button iconName="trash-can"> </app-button>
```

Comment:
`iconName` is an input for ButtonComponent. Can you please update to reflect the appropriate input notation?

Reply:
Good point, done!